Assignment4 last changes

anamarine opened the pull request:
Saved .nt with UTF-8 encoding and created .sparql with queries

anamarine committed:
.nt to utf-8 encoding

anamarine committed:
Merge branch 'assignment4' of https://github.com/Lucasct77/Curso2020-2021-ODKG into assignment4

anamarine committed:
Create actividad_contractual.nt

anamarine committed:
Sparql queries

brodrigol committed:
Update queries.sparql

Commit:
- line breaks deleted

Commit:
Merge branch 'master' of https://github.com/Lucasct77/Curso2020-2021-ODKG into assignment4

# Conflicts:
#	HandsOn/Group02/csv/juntas_distrito_ayuto_madrid-Updated.csv

Commit:
- ansi to utf8

Lucasct77 commented on HandsOn/Group02/mappings/actividad_contractual_mapping.ttl:
```diff
@@ -0,0 +1 @@
+];
```
Este fichero solo tiene esto, entiendo que hay que borrarlo, no?

Commit:
Merge branch 'master' of https://github.com/Lucasct77/Curso2020-2021-ODKG into assignment4

anamarine committed:
Updates in ontologies and mappings

anamarine committed:
Changes in mappings

Changed object names in mappings so it's coherent with blanca's last version of the juntas_de_distrito.csv
Also, added object types (which were missing in the art of the juntas de distrito)

Commit:
- new nt
- deleted old property

Commit:
- change property9 to hasSection
